#BEGIN: [Tue Sep 26 13:10:08 2023]
#HOST: s02
#TIMELIMIT: 1800
#MEMLIMIT: 0
#SOLVER: existent-cpu-suisen-cp2
#INSTANCE1: /benchmark/random_instance/random_instance040_graph001.col
#INSTANCE2: /benchmark/random_instance/random_instance040_graph001_01.dat
#COMMAND: docker run --cidfile /tmp/cid.txt --init --rm -t --ulimit cpu=1800 -v /home/share/benchmark/ISR-benchmark:/benchmark suisen-cp2 /benchmark/random_instance/random_instance040_graph001.col /benchmark/random_instance/random_instance040_graph001_01.dat
[36m04:10:16 [INFO] Time reorder   : 7 (sec)[0m
Traceback (most recent call last):
  File "/solver-dir/./Main.py", line 174, in <module>
    solve(
  File "/solver-dir/./Main.py", line 77, in solve
    source = [mapping[u] for u in source]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/solver-dir/./Main.py", line 77, in <listcomp>
    source = [mapping[u] for u in source]
              ~~~~~~~^^^
KeyError: 289
Command exited with non-zero status 1
	Command being timed: "bash /solver-dir/run.sh /benchmark/random_instance/random_instance040_graph001.col /benchmark/random_instance/random_instance040_graph001_01.dat"
	User time (seconds): 7.80
	System time (seconds): 0.04
	Percent of CPU this job got: 99%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:07.90
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 53404
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 102
	Minor (reclaiming a frame) page faults: 16606
	Voluntary context switches: 610
	Involuntary context switches: 27
	Swaps: 0
	File system inputs: 35960
	File system outputs: 32
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 1
#END: [Tue Sep 26 13:10:16 2023]
